### PR TITLE
[FIX] ECAT8 Norm only checked one of three possible scanner names.

### PR DESCRIPTION
**Pull request by ashgillman**
originating systems accepted are "2008", "Simens mMR" and "mMR"

### CODE REVIEW

File: src/recon_buildblock/BinNormalisationFromECAT8.cxx [hunk 1]
```diff
@@ -282,10 +282,13 @@ MatrixFile* mptr = matrix_open(filename.c_str(),  MAT_READ_ONLY, Norm3d);
   s.erase( std::remove_if( s.begin(), s.end(), isspace ), s.end() );
   /*interfile_parser.*/data_file_name=s;
   
-  if (/*interfile_parser.*/originating_system == "2008")
+  if (/*interfile_parser.*/originating_system == "2008"
+      or originating_system == "SiemensmMR" // we stripped the space
```
Comment (KrisThielemans) [inline src/recon_buildblock/BinNormalisationFromECAT8.cxx]:
```suggestion
      || (originating_system == "SiemensmMR") // we stripped the space
```
C++11? fails on [Appveyor](https://ci.appveyor.com/project/KrisThielemans/stir/builds/20076321/job/kw1tosgov218mcbn)
I feel safer with `()`
similar elsewhere.

Comment (KrisThielemans) [reply src/recon_buildblock/BinNormalisationFromECAT8.cxx]: by the way, why stripped the space?

Comment (ashgillman) [reply src/recon_buildblock/BinNormalisationFromECAT8.cxx]: Haha, ah can you tell what language I've been using all day?

The stripped space was done a few lines prior, apparently to remove trailing newline (specifically \r).

**Comment by ashgillman** [comment]
> Does this actually occur in Siemens files?

Hmm, no. So maybe somewhere was trying to load my norm sinogram as ECAT8 and not catching the error?

**Comment by KrisThielemans** [comment]
ah. if you have a norm sinogram, then this will never work of course. you need the "from projdata" `BinNormalisation`.

This hierarchy doesn't (yet) discover what the type is. you have to specify the class by hand.

**Comment by KrisThielemans** [comment]
If this doesn't occur in the Siemens file, I'm not sure if we should accept this.

**Comment by ashgillman** [comment]
This makes sense - but then why did this change make my script progress (at least to the next bug)? Would SIRF somewhere first try ECAT8, then if it fails, try sinogram, for example when making a sensitivity model?

**Comment by ashgillman** [comment]
I agree

**Comment by ashgillman** [comment]
I'll have another explore

**Comment by KrisThielemans** [comment]
> why did this change make my script progress (at least to the next bug)? Would SIRF somewhere first try ECAT8, then if it fails, try sinogram, for example when making a sensitivity model?

STIR no, SIRF probably...

**Pull request closed without merging**